src/LinAlg/hiopLinSolverIndefDenseMagma.cpp

```diff
@@ -119,9 +136,40 @@ namespace hiop
     }
     //printf("(pos,null,neg)=(%d,%d,%d)\n", posEigVal, nullEigVal, negEigVal);
     nlp_->runStats.linsolv.tmInertiaComp.stop();
-    
-    if(nullEigVal>0) return -1;
-    return negEigVal;
+
+    return true;
+  }
+
+  bool hiopLinSolverIndefDenseMagmaBuKaDev::compute_inertia(double **MM, 
+							 int N, 
+							 int *ipiv,
+							 int& posEigVal, 
+							 int& negEigVal, 
+							 int& nullEigVal)
+  {
+    double det[2];
+    int inert[3];
+    int job = 110;
+    int info;
+
+    nlp_->runStats.linsolv.tmInertiaComp.start();
+
+    if(NULL==work_) {
+      work_ = new double[N];
+    }
+
+
+    //determinant if only for logging/output/curiosity purposes
+    magma_dsidi(MM[0], N, N, ipiv, det, inert, work_, job, &info);
```

Comment:
```suggestion
    magma_dsidi(MM, N, N, ipiv, det, inert, work_, job, &info);
```
If `MM` is `double*` type we do not need to dereference here.

Reply:
@ashermancinelli and I are pm'ed about this recently as I am looking to solve issue #81.